src/uct/api/uct.h

```diff
@@ -265,6 +265,25 @@ enum uct_iface_event_types {
 
 /**
  * @ingroup UCT_RESOURCE
+ * @brief  Flush modifiers.
+ */
+enum uct_flush_flags {
+    UCT_FLUSH_FLAG_LOCAL    = 0,            /**< Guarantees that the data
+                                                 transfer is completed but the
+                                                 target buffer may not be
+                                                 updated yet.*/
```

Comment:
What is completed means here ? Can you please comment what it really means ?

Reply:
It's equal to only one existing "flush" mode before and copy-pasted from: https://github.com/openucx/ucx/pull/1794/files#diff-a1f319ca66018bb8d03274d3b1757ec7L1437
Forgot to remove this note...